### PR TITLE
Conference room shutters

### PR DESCRIPTION
**Pull request by OolongCow**
As suggested by [niennab](https://forums.aurorastation.org/topic/8983-minor-mapping-suggestions/page/9/#comment-165160), I've added shutters that start open to the bridge conference room for the sake of CCIA interviews. The button is by the aft door light switch.

shutters

### CODE REVIEW

**Comment by OolongCow** [comment]
!review

**Comment by OolongCow** [comment]
SDMM sanitizes a lot of variables, which is why you may see some unrelated code deletions.

**Review by niennab — APPROVED** [review]

File: html/changelogs/example copy.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Adds shutters (by default opened) to the bridge conference room as suggested by niennab."
```
Comment (SleepyGemmy) [inline html/changelogs/example copy.yml:41]:
```suggestion
  - maptweak: "Adds shutters (by default opened) to the bridge conference room."
```

Comment (OolongCow) [reply html/changelogs/example copy.yml]: True. I was sleepy when I wrote that.

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**